src/main/kotlin/com/intershop/gradle/icm/extension/DevelopmentConfiguration.kt

```diff
@@ -146,23 +123,6 @@ open class DevelopmentConfiguration
         }
     }
 
-    /**
```

Comment:
Please remove all the license configuration related documentation from README too.